docs/en/agent/support-plugins.md

```diff
@@ -11,9 +11,11 @@ metrics based on the tracing data.
   * `fasthttp`: [FastHttp](https://github.com/valyala/fasthttp) tested v1.10.0 to v1.50.0.
   * `fiber`: [Fiber](https://github.com/gofiber/fiber) tested v2.49.0 to v2.50.0.
   * `echov4`: [Echov4](https://github.com/labstack/echo) tested v4.0.0 to v4.11.4
+  * `goframe`: [GoFrame](https://github.com/gogf/gf/) tested v2.6.0 to v2.7.3
 * HTTP Client
   * `http`: [Native HTTP](https://pkg.go.dev/net/http) tested go v1.19 to go v1.23.
   * `fasthttp`: [FastHttp](https://github.com/valyala/fasthttp) tested v1.10.0 to v1.50.0.
+  * `goframe`: [GoFrame](https://github.com/gogf/gf/) tested v2.6.0 to v2.7.3
```

Comment:
Looks like you just implement the server side plugin?

Reply:
I think it's okay not to write it here,
 because as I mentioned earlier,
 goframe  httpclient is wraps net/http

Reply:
I remove this change